app/src/pages/project/SessionsTable.tsx

```diff
@@ -93,7 +101,10 @@ export function SessionsTable(props: SessionsTableProps) {
       props.project
     );
   const tableData = useMemo(() => {
-    return data.sessions.edges.map(({ session }) => session);
+    return data.sessions.edges.map(({ session }) => ({
+      ...session,
+      tokenCountTotal: session.tokenUsage.total,
+    }));
   }, [data]);
```

Comment:
```suggestion
  }, [data.sessions]);
```

Reply:
Not sure if this makes a difference, but might be worth a try.

Reply:
it probably won't in this case but doesn't hurt

Reply:
ok. will change. thx

Reply:
either way is all good!